### PR TITLE
Generic CSV Import

### LINKED ISSUE #2270: Generic CSV import template + Optional platform specific formats
## Abstract
First time users of Rotki are coming from a variety of tax reporting/trading tools which (mostly) export to CSV and new platforms are popping up every day.  

To support the widest range of new users as possible, Rotki should establish its own CSV import format so users can manually format the data themselves which would allow importing _from any platform_.  Once completed, platform specific import features can be created for the top platforms.

## Motivation
**Problem** - Currently, users with abundant trade history on other platforms (bitcoin.tax for example) can't switch to Rotki unless their platform is supported for data import.

**Solution** - Offer a generic CSV import template so users can format the data themselves.  This enables importing from _any_ platform.

## Specification
On the Import Data screen, you would have 2 options:
1. Select a supported platform to import from
2. If your platform is not supported (yet) download _rotki_trade_history_template.csv_ which has pre-defined column headers so you can format the data yourself and import

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #2270

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/import/CustomImport.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <v-row>
+      <v-col>
+        <import-source source="rotki_events">
+          <template #upload-title>
+            <strong>
+              {{ $t('import_data.custom.import_events') }}
```
Comment (kelsos) [inline frontend/app/src/components/import/CustomImport.vue:8]: @lukicenturi for new components please `<script setup>` and also 
make sure to use `t` and `tc` from `useI18n()`,

In general, avoid using `$` prefixed properties and instead use whatever alternative exists in vue 2.7.

If we don't do this we end up giving us more work in the future.

I was looking for a general rule for `$` prefixed globals but couldn't find anything so we have to keep it in mind.

File: frontend/app/src/components/import/GroupedImport.vue [hunk 2]
```diff
@@ -39,56 +47,72 @@ import { get } from '@vueuse/core';
...
+import BinanceImport from '@/components/import/BinanceImport.vue';
+import BisqImport from '@/components/import/Bisq.vue';
+import BlockFiImport from '@/components/import/BlockFiImport.vue';
+import CointrackingImport from '@/components/import/CointrackingImport.vue';
+import CryptoComImport from '@/components/import/CryptoComImport.vue';
+import CustomImport from '@/components/import/CustomImport.vue';
+import NexoImport from '@/components/import/NexoImport.vue';
+import ShapeshiftImport from '@/components/import/ShapeshiftImport.vue';
+import UpholdImport from '@/components/import/UpholdImport.vue';
+import i18n from '@/i18n';
```
Comment (kelsos) [inline frontend/app/src/components/import/GroupedImport.vue:59]: In this components `vue-i18n-composable` is already used, please don't import `@/i18n` directly but use the `tc` or `t` from from `useI18n`

**Review by kelsos — COMMENTED** [review]
@lukicenturi if we fix the two comments I left I think we can merge it.

File: frontend/app/src/components/import/GroupedImport.vue (hunk 2 above)
Comment (lukicenturi) [reply frontend/app/src/components/import/GroupedImport.vue]: Sorry, I think I missed it when rebasing

**Review by kelsos — APPROVED** [review]
lgtm, thank you @lukicenturi

**Pull request merged**